parachain/pallets/ethereum-beacon-client/src/config.rs

```diff
@@ -1,17 +1,39 @@
-pub const MAX_PROPOSER_SLASHINGS: usize = 16;
+#[cfg(feature = "minimal")]
+mod minimal;
 
-pub const MAX_ATTESTER_SLASHINGS: usize =  2;
+#[cfg(not(feature = "minimal"))]
+mod mainnet;
```

Comment:
If we name the config files per runtime(see comment below) we can re-use the existing features `snowbase-native`, `snowblink-native` and `snowbridge-native` to choose the appropriate config for each runtime.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/661/commits/b6a353d0049de0d2fc379e28fd99c5c96a43ef37.